### PR TITLE
Add documentation on how to use an external GPS clock

### PR DESCRIPTION
**Pull request by dsilhavy**
Adds a short description on how to use an external GPS clock for the 5G setup

### CODE REVIEW

File: pages/5g-core-network-components/tutorials/5gnetwork.md [hunk 1]
```diff
@@ -10,17 +10,21 @@ nav_order: 0
 # Tutorial - 5G Network Setup with COTS UE
 
 ## Introduction
-These are the generic instructions to setup a 5G network using Open5GS and srsRAN. An Ettus X310 USRP and a Pixel 8 phone are used. 
+
+These are the generic instructions to setup a 5G network using Open5GS and srsRAN. An Ettus X310 USRP and a Pixel 8
```
Comment (rjb1000) [inline pages/5g-core-network-components/tutorials/5gnetwork.md:14]: Should be "to set up" ("setup" is only one word when used as a noun).

File: pages/5g-core-network-components/tutorials/5gnetwork.md [hunk 2]
```diff
@@ -293,10 +308,37 @@ pcap:
   ngap_filename: /tmp/gnb_ngap.pcap   # Path where the NGAP PCAP is stored.
 ```
 
-## Running the 5G Core (Open5GS)
+### Optional: Adding an external GPS reference clock
+
+Ideally the SDR (USRPs) are connected to a 10 MHz external reference clock or GPSDO, although this is not a strict
+requirement. In our tests, many COTS UEs were only able to connect to the gNB without the use of an external reference
```
Comment (rjb1000) [inline pages/5g-core-network-components/tutorials/5gnetwork.md:314]: Something not quite right here. 
> In our tests, many COTS UEs were only able to connect to the gNB without the use of an external reference clock.

Either:

> In our tests, many COTS UEs were only able to connect to the gNB **with** the use of an external reference clock.

or:

> In our tests, many COTS UEs were ~~only~~ able to connect to the gNB without the use of an external reference clock.

Not sure which you meant!

**Review by rjb1000 — CHANGES_REQUESTED** [review]
Just a couple of queries on the documentation.

File: pages/5g-core-network-components/tutorials/5gnetwork.md (hunk 1 above)
Comment (dsilhavy) [reply pages/5g-core-network-components/tutorials/5gnetwork.md]: Thanks, fixed.

File: pages/5g-core-network-components/tutorials/5gnetwork.md (hunk 2 above)
Comment (dsilhavy) [reply pages/5g-core-network-components/tutorials/5gnetwork.md]: Thanks, changed to:

> In our tests, many COTS UEs were only able to connect to the gNB with the use of an external reference
clock.

**Comment by jordijoangimenez** [comment]
Let's merge. Some minor update.

**Pull request merged**